birdhouse/components/geoserver/.gitignore

```diff
@@ -6,4 +6,3 @@ service-config.json
 geoserver_canarie_api_monitoring.py
 geoserver-magpie-provider.cfg
 config/proxy/canarie_api_monitoring.py
-config/magpie/providers.cfg
```

Comment:
Geoserver do not add to magpie providers anymore?  But should we still keep this old ignore to avoid breaking autodeploy?

Reply:
We need to remove this so that future changes to the providers.cfg file won't be ignored. It is no longer generated from a template file, it is part of the source code now.